ui/ts/components/navbar.ts

```diff
@@ -31,18 +32,19 @@ module Components {
     export function controller(ts: TargetSet): any {}
 
     export function view(ctrl: any, ts: TargetSet): _mithril.MithrilVirtualElement {
-      return m("ul.navigation", _.map(ts.targets(), (t: Target) =>
-        m("li",
-          {
-            className: ts.isActive(t) ? "active" : ""
-          },
-          m("a",
+      return m("ul.navigation-sidebar", _.map(ts.targets(), (t: Target) =>
+          m("li",
             {
-              config: m.route,
-              href: ts.baseRoute + t.route
+              className: ts.isActive(t) ? "active" : ""
             },
-            t.title)
-         )
+            m("a",
+              {
+                config: m.route,
+                href: ts.baseRoute + t.route
+              },
+              _.isString(t.view) ? m("div", t.view) : t.view
```

Comment:
No need for this, m() can already accept any of the types t.view is allowed to be.  No need to wrap the string in an additional div.